dev-ostelco-ios-clientTests/CoordinatorTests/NorwayUserHappyFlowWithJumioStageDeciderTests.swift

```diff
@@ -25,7 +25,7 @@ class NorwayUserHappyFlowWithJumioStageDeciderTests: XCTestCase {
     
     func testUserHasCompletedJumio() {
         let decider = StageDecider()
-        let localContext = LocalContext(hasSeenNotificationPermissions: true, hasSeenVerifyIdentifyOnboarding: true, hasCompletedJumio: true, hasSeenLocationPermissions: true)
+        let localContext = LocalContext(hasSeenNotificationPermissions: true, hasCompletedJumio: true, hasSeenLocationPermissions: true)
```

Comment:
Much repeated code her.  Is it possible to delegate this either to a method or to the fixture setup for the test suite?

Reply:
Created a task in notion to address this

Reply:
This isn't repeated code, it's just the data the test uses. There's no logic being duplicated here. Fixtures would only make this even more complicated.